src/mvPlotting.cpp

```diff
@@ -2376,13 +2436,13 @@ DearPyGui::set_configuration(PyObject* inDict, mvPlotConfig& outConfig)
 	flagop("no_title", ImPlotFlags_NoTitle, outConfig._flags);
 	flagop("no_menus", ImPlotFlags_NoMenus, outConfig._flags);
 	flagop("no_box_select", ImPlotFlags_NoBoxSelect, outConfig._flags);
-	flagop("no_mouse_pos", ImPlotFlags_NoMousePos, outConfig._flags);
-	flagop("no_highlight", ImPlotFlags_NoHighlight, outConfig._flags);
-	flagop("no_child", ImPlotFlags_NoChild, outConfig._flags);
-	flagop("query", ImPlotFlags_Query, outConfig._flags);
+	flagop("no_mouse_text", ImPlotFlags_NoMouseText, outConfig._flags);
 	flagop("crosshairs", ImPlotFlags_Crosshairs, outConfig._flags);
-	flagop("anti_aliased", ImPlotFlags_AntiAliased, outConfig._flags);
 	flagop("equal_aspects", ImPlotFlags_Equal, outConfig._flags);
+	flagop("no_legend", ImPlotFlags_NoLegend, outConfig._flags);
```

Comment:
Do we need an explicit argument for this?

The old version controls legend visibility by showing or hiding the `plot_legend` widget; if there's no `add_plot_legend`, the legend is hidden by default - which looks reasonable. The new version also switches `ImPlotFlags_NoLegend` when `plot_legend` gets shown or hidden. I'd say `config.show` on `plot_legend` is sufficient, whereas having more than one way to control this `ImPlotFlags_NoLegend` flag is excessive and will eventually lead to inconsistency.

Reply:
It makes sense, I'll remove it